lint.sh

```diff
@@ -24,7 +24,10 @@ if [ -n "$index_file" ]; then
     cd /tmp
     sort index-body.md > index-body-sorted.md
     diff index-body.md index-body-sorted.md
+    package_name=$(awk '{print $1}' index-body.md)
+    for i in $package_name; do
+        expr "$i" : "[a-z0-9\.-]\+$" > /dev/null || (echo "$i: Product names shall be all in lower case" && exit 250)
```

Comment:
英文不规范：术语应该和前面一致用 package，而且这里应该用单数，而复数显然不对：

> Chrome: Product names shall be all in lower case

```suggestion
        expr "$i" : "[a-z0-9\.-]\+$" > /dev/null || (echo "$i: Package name shall be all in lower case" && exit 250)
```